### PR TITLE
Use multi-root aware diagram-js

### PR DESCRIPTION
**Pull request by nikku**
Related to https://github.com/bpmn-io/diagram-js/pull/601.

### CODE REVIEW

**Comment by nikku** [comment]
@marstamm To be examined.

File: lib/import/Importer.js [hunk 1]
```diff
@@ -76,7 +76,12 @@ export function importBpmnDiagram(diagram, definitions, bpmnDiagram) {
     }
 
     var mainDiagram = bpmnDiagram || definitions.diagrams[0];
-    canvas.setActivePlane(mainDiagram.plane.bpmnElement.id);
+
+    canvas.setRootElement(
```
Comment (nikku) [inline lib/import/Importer.js:80]: Question to you @marstamm: How do we currently import a process + drill down right away?

I guess we don't support case, but rather import a single diagram only when it is being passed to the bpmn-js instance via `#open`:

```
await bpmnJs.open(diagramXML, 'ID_OF_DIAGRAM');
```

Comment (nikku) [reply lib/import/Importer.js:80]: If we did support importing all + opening a single diagram, we'd need to change this line below :arrow_down:.

Comment (marstamm) [reply lib/import/Importer.js:80]: No, we do not support it at this point. However, it is on our roadmap: https://github.com/bpmn-io/bpmn-js/issues/1546

Comment (nikku) [reply lib/import/Importer.js:85]: In this case my proposal is to make importing future proof already: https://github.com/bpmn-io/bpmn-js/pull/1551/commits/67f42f53492c416ebf282cf80e95bce5e3002d67

**Comment by nikku** [comment]
Pending https://github.com/bpmn-io/diagram-js/pull/601 to be asserted and merged.

File: lib/features/drilldown/DrilldownOverlays.js [hunk 2]
```diff
@@ -107,4 +101,13 @@ var getParentChain = function(child) {
...
+}
+
+
+function planeId(element) {
+  if (is(element, 'bpmn:SubProcess')) {
+    return element.id + '_plane';
+  }
+
+  return element.id;
+}
```
Comment (marstamm) [inline lib/features/drilldown/DrilldownOverlays.js:113]: Would it make sense to expose this function in a utility Function? We will probably need it in other places as well.

File: lib/import/Importer.js [hunk 3]
```diff
@@ -76,7 +76,15 @@ export function importBpmnDiagram(diagram, definitions, bpmnDiagram) {
...
     var mainDiagram = bpmnDiagram || definitions.diagrams[0];
-    canvas.setActivePlane(mainDiagram.plane.bpmnElement.id);
+
+    var rootId = mainDiagram.plane.bpmnElement.id;
+
+    // we do need to account for different ways we create root elements
+    // each nested imported <root> do have the `_plane` suffix, while
+    // the root <root> is found under the business object ID
+    canvas.setRootElement(
+      canvas.findRoot(rootId + '_plane') || canvas.findRoot(rootId)
```
Comment (marstamm) [inline lib/import/Importer.js:86]: Here we could use the `planeId` utility

File: lib/features/drilldown/DrilldownOverlays.js (hunk 2 above)
Comment (nikku) [reply lib/features/drilldown/DrilldownOverlays.js:113]: We _could_ expose it. However, this is not a safe operation _per se_. If we render the sub-process as a root (only element imported) this naming assumption does not longer hold. 

TLDR: I'd love to continue to improve this in the future, if we have the full picture, not now.

File: lib/import/Importer.js (hunk 3 above)
Comment (nikku) [reply lib/import/Importer.js:86]: Unfortunately we cannot use the utility here. How to find the root depends on whether we import the sub-process as the root of the diagram (it is called `subprocess(nosuffix)` then or as a nested sub-process plane, called `subprocess_plane` then).

Comment (nikku) [reply lib/import/Importer.js:86]: => It is safer to look it up searching for plane first, and actual element next.

File: lib/features/drilldown/DrilldownOverlays.js (hunk 2 above)
Comment (marstamm) [reply lib/features/drilldown/DrilldownOverlays.js:113]: Makes sense. Lets not do it now and expose it when we need it :+1:

File: lib/import/Importer.js (hunk 3 above)
Comment (marstamm) [reply lib/import/Importer.js:86]: Makes sense with the current semantic of `open`. 
We should revisit this when we tackle https://github.com/bpmn-io/bpmn-js/issues/1546, as "subProcess is root only" should not exist after that

**Comment by nikku** [comment]
Please give your approval. We can iterate on the details in the course of time :ship:

**Review by marstamm — APPROVED** [review]
